UI/Pages/SearchModel.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
 using System.Text.RegularExpressions;
+using Google.Apis.Drive.v3.Data;
```

Comment:
I don't think we should couple the search model to a Google API class.

Reply:
Removed